.changeset/odd-countries-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@frontify/fondue': patch
+---
+
+Fix error when scrolling in modal
```

Comment:
`fix: <description of what it solves>`

Try to be a bit more precise than "error"